### PR TITLE
Remove propTypes in production

### PR DESCRIPTION
**Pull request by TrySound**
This saves ~7kB

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "presets": ["jason"],
+  plugins: [
+    ['transform-react-remove-prop-types', { mode: 'wrap' }],
```
[the remark this answers is not in the document]
Comment (TrySound) [reply .babelrc]: Isn't user validation one of the points of having prop types?

[the remark this answers is not in the document]
Comment (TrySound) [reply .babelrc]: It will save only 300b. So we may skip it for now

[the remark this answers is not in the document]
Comment (jquense) [reply .babelrc]: ah right, it'd be removed for the unminified build as well

[the remark this answers is not in the document]
Comment (TrySound) [reply .babelrc]: Prop types are not removed. This project has two blockers mixing and old context api.

**Comment by TrySound** [comment]
@jquense Is this ok for merge?

**Pull request merged**